clevertap-pushtemplates/src/main/java/com/clevertap/android/pushtemplates/content/PendingIntentFactory.kt

```diff
@@ -183,71 +178,17 @@ internal object PendingIntentFactory {
                 }
             }
 
-            RATING_CLICK1_PENDING_INTENT -> {
+            RATING_CLICK1_PENDING_INTENT, RATING_CLICK2_PENDING_INTENT, RATING_CLICK3_PENDING_INTENT,
+            RATING_CLICK4_PENDING_INTENT, RATING_CLICK5_PENDING_INTENT -> {
+                val clickedStar = identifier - RATING_CLICK1_PENDING_INTENT + 1
```

Comment:
make a function to return **int** based on **identifier** rather than having a magic number function like this.